src/irc.py

```diff
@@ -96,11 +95,11 @@ def prepare(self) -> None:
             logger.info("Created directory %s", IRC_BRIDGE_CONFIG_DIR_PATH)
             shutil.copy(IRC_BRIDGE_TEMPLATE_CONFIG_FILE_PATH, IRC_BRIDGE_CONFIG_DIR_PATH)
 
-        if not IRC_BRIDGE_UNIT_FILE_PATH.exists() or not IRC_BRIDGE_TARGET_FILE_PATH.exists():
-            shutil.copy(IRC_BRIDGE_TEMPLATE_UNIT_FILE_PATH, SYSTEMD_DIR_PATH)
-            shutil.copy(IRC_BRIDGE_TEMPLATE_TARGET_FILE_PATH, SYSTEMD_DIR_PATH)
-            systemd.daemon_reload()
-        systemd.service_enable(IRC_BRIDGE_SNAP_NAME)
+        with open("/etc/environment", "r+", encoding="utf-8") as env_file:
```

Comment:
Is it not necessary to reload the snap after that?

Reply:
This is the preparation phase, tje snap is not started yet.